tests/unit/parsers/test.manifestjson.js

```diff
@@ -96,6 +96,33 @@ describe('ManifestJSONParser', () => {
     expect(warnings[0].code).toEqual('IGNORED_APPLICATIONS_PROPERTY');
   });
 
+  it('should fail if "homepage_url" links to AMO domain names', () => {
+    const addonLinter = new Linter({ _: ['bar'] });
+    const json = validManifestJSON({
+      homepage_url: 'http://addons.mozilla.org',
```

Comment:
we may want to use an url with a path here to also verify as a side-effect that we are not just checking for strictly equality.

To be totally fair, we could provide even better coverage if we do turn these two test cases into a parametrized function that we will use to test more scenarios, something like:

```js
describe('homepage_url', () => {
  function testHomepageUrl(homepage_url, expectValid) {
    ...
    const json = validManifestJSON({ homepage_url });
    ...
    expect(manifestJSONParser.isValid).toEqual(expectValid);
    if (expectValid) {
      // expect errors.length is 0 
    } else {
      // expect errors.length is 1 and the error code the expected one
    }
  }

  const testInvalidUrls = [
    'https://addons.mozilla.org',
    'http://addons.mozilla.org/somepath',
  ];
  for (let invalidUrl of testInvalidUrls) {
    it(`should fail on forbidden homepage_url "${invalidUrl}"`, () => {
      return testHomepageUrl(invalidUrl, false);
    });
  }

  it('should not mark as invalid on homepage_url using a restricted domain', () => {
    return testHomepageUrl("http://test.org", true);
  });
```

Reply:
@rpl Done, sorry for keeping forgetting and using AMO :laughing:.
Let me know about any further changes needed.